doc/sphinx/Clusters_from_Scratch/shared-storage.rst

```diff
@@ -29,17 +29,18 @@ repository:
 .. code-block:: none
 
     # rpm --import https://www.elrepo.org/RPM-GPG-KEY-elrepo.org
-    # rpm -Uvh http://www.elrepo.org/elrepo-release-7.0-3.el7.elrepo.noarch.rpm
-    Retrieving http://www.elrepo.org/elrepo-release-7.0-3.el7.elrepo.noarch.rpm
+    # rpm -Uvh https://www.elrepo.org/elrepo-release-8.el8.elrepo.noarch.rpm
+    Retrieving https://www.elrepo.org/elrepo-release-8.el8.elrepo.noarch.rpm
+    Verifying...                          ################################# [100%]
     Preparing...                          ################################# [100%]
     Updating / installing...
-       1:elrepo-release-7.0-3.el7.elrepo  ################################# [100%]
+       1:elrepo-release-8.2-1.el8.elrepo  ################################# [100%]
```

Comment:
I noticed an unrelated spelling error at the start of this chapter: "care" should be "can"